### PR TITLE
Doc: disable-outside-detected-project

### PR DESCRIPTION
**Pull request by gpetiot**
Shoud address issue #456

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,12 @@ Alternately, see [`ocamlformat.opam`](./ocamlformat.opam) for manual build instr
 
 ## Editor setup
 
+### Disable outside project
+
+As mentioned in the Options section, when the option `--disable-outside-detected-project` is set, .ocamlformat files outside of the project (including the one in `XDG_CONFIG_HOME`) are not read. The project root of an input file is taken to be the nearest ancestor directory that contains a .git or .hg or dune-project file. If no config file is found, then the formatting is disabled.
+
+This feature is often the behavior you can expect form OCamlFormat when it is directly run from your text editor, so it is advised to use this option.
```
Comment (jberdine) [inline README.md]: s/form/from

File: README.md [hunk 2]
```diff
@@ -117,12 +123,30 @@ Alternately, see [`ocamlformat.opam`](./ocamlformat.opam) for manual build instr
   (add-hook 'before-save-hook 'ocamlformat-before-save)))
 ```
 
+To change the options passed to OCamlFormat (to use the option `--disable-outside-detected-project` for example):
+- run `emacs`
+- run `M-x customize`
+- enter "ocamlformat" in the search field, and press the Search button
+- select the Ocamlformat Command item
+- enter your customized command in the Ocamlformat Command field, for example `ocamlformat --disable-outside-detected-project`
```
Comment (jberdine) [inline README.md]: Have you tested this, to make sure that this field is interpreted as a command rather than an executable name?

Comment (gpetiot) [reply README.md]: You are right I added a choice option in the customization menu instead.

File: README.md [hunk 3]
```diff
@@ -117,12 +123,32 @@ Alternately, see [`ocamlformat.opam`](./ocamlformat.opam) for manual build instr
   (add-hook 'before-save-hook 'ocamlformat-before-save)))
 ```
 
+To pass the option `--disable-outside-detected-project` (or `--disable`) to OCamlFormat:
+- run `emacs`
+- run `M-x customize`
+- enter "ocamlformat" in the search field, and press the Search button
```
Comment (jberdine) [inline README.md]: maybe `M-x customize-group⏎` then enter `ocamlformat⏎`

**Pull request merged**